src/main/java/de/exercise/bisect/MainApp.java

```diff
@@ -52,4 +54,8 @@ private static Double bisect(double a, double b) {
         return m;
     }
 
+    public static int sum(int z1, int z2) {
+        return z1 + z2;
```

Comment:
was ist z1 und z2 ?

Reply:
ja stimmt. Doof von mir. Die Variablen stehen für Zahlen. Ich benenne sie lieber in Zahl statt z.